docs/source/concepts/include/snippets/workflows/workchains/run_workchain_submit_parallel.py

```diff
@@ -15,9 +15,9 @@ def submit_workchains(self)
         for i in range(3):
             future = self.submit(SomeWorkChain)
             key = 'workchain_{}'.format(i)
-            self.to_context(key=future)
+            self.to_context(**{key:future})
```

Comment:
Why did you change this notation? The decomposition will be exactly what it was before, no?